utilities/cheyenne/launch-dask.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os, sys, argparse, subprocess, re, time, logging, socket
+from dask.distributed import Client
+logger = logging.getLogger(__name__)
+
+def parse_command_line(args, description):
+    parser = argparse.ArgumentParser(description=description,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--project",default=os.environ.get("PROJECT"),
+                        help="Specify a project id for the case (optional)."
+                        "Used for accounting when on a batch system."
+                        "The default is user-specified environment variable PROJECT")
+    parser.add_argument("--workers", default=0, type=int,
+                        help="Number of client nodes to launch")
+    parser.add_argument("--walltime", default="01:00:00",
+                        help="Set the wallclock limit for all nodes. ")
+
+    parser.add_argument("--workdir",
+                        default=os.path.join("/glade","scratch",os.environ.get("USER")),
+                        help="Set the working diirectory")
+
+    parser.add_argument("--notebookdir",
+                        default=os.path.join("/glade","p","work",os.environ.get("USER")),
+                        help="Set the working diirectory")
+
```

Comment:
It would also be good to configure which queue these jobs land on, as well as the breakdown of processes/threads. Also the notebook and dask-dashboard ports should be configurable.